### PR TITLE
Added two additonal functions for Xml Writer

### PR DESCRIPTION
**Pull request by angela2389**
Hi Microsoft team,

Can you please add these two new functions for Xml Writer, for both BC16 and BC17?

### CODE REVIEW

File: Modules/System/XmlWriter/src/XmlWriterImpl.Codeunit.al [hunk 1]
```diff
@@ -19,6 +19,12 @@ codeunit 1484 "XmlWriter Impl."
         XmlTextWriter.WriteStartDocument();
     end;
 
+    procedure WriteProcessingInstruction(Encoding: Text)
+    begin
+        Initialize();
+        XmlTextWriter.WriteProcessingInstruction('xml', STRSUBSTNO('version="1.0" encoding="%1" standalone="no"', Encoding));
```
Comment (mazhelez) [inline Modules/System/XmlWriter/src/XmlWriterImpl.Codeunit.al:25]: Why not implement the actual [WriteProcessingInstruction](https://docs.microsoft.com/en-us/dotnet/api/system.xml.xmlwriter.writeprocessinginstruction?view=net-5.0)?

Something like 
```
WriteProcessingInstruction(Name: Text, "Text": Text)
```

Comment (angela2389) [reply Modules/System/XmlWriter/src/XmlWriterImpl.Codeunit.al]: Yes, that's better indeed. I have updated the code!

**Review by mazhelez — APPROVED** [review]
Thanks for your contribution, @angela2389

**Comment by JesperSchulz** [comment]
Work for 16 and 17 pending, but in progress.

**Comment by bc-ghost** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. We will update this issue with information about availability.

**Comment by bc-ghost** [comment]
Thanks for submitting this issue. We will publish a fix for it in the next minor update for release 17.

**Comment by bc-ghost** [comment]
Thanks for submitting this issue. We will publish a fix for it in the next minor update for release 16.

**Pull request merged**